generators/client/templates/react/src/main/webapp/app/reducers/authentication.ts

```diff
@@ -123,8 +123,8 @@ export const login = (username, password, rememberMe = false) => async (dispatch
   dispatch(getSession());
 };
 
-export const logout = () => async dispatch => {
-  await dispatch({
+export const logout = () => dispatch => {
+  dispatch({
```

Comment:
Seems like for logout this api call is not required, I'm not sure why I added this originally